Health/src/main/webapp/app/controller/dataframe/CancerData.js

```diff
@@ -20,31 +20,108 @@ function getColumnName(){
 	var DataFrame = dfjs.DataFrame;
```

Comment:
Okay, but that's a reverse engineering.
It will be better if we modify them in classes in early stages, later it will be more difficult.
lets schedule a meeting today and lets do it.